elkm1_lib/keypads.py

```diff
@@ -61,6 +79,13 @@ def _kc_handler(self, keypad: int, key: int) -> None:
             name = ""
         self.elements[keypad].setattr("last_keypress", (name, key), True)
 
+    def _kf_handler(self, keypad: int, key: str, chime_mode: list[int]):
+        try:
+            name = FunctionKeys(key).name
+        except ValueError:
+            name = ""
```

Comment:
Similar comment as before. Passing `name = ""` probably not right thing to do.

Reply:
works like kc_decode, let me know if you want all to change to work differently?